nutanix/data_source_nutanix_cluster.go

```diff
@@ -2,20 +2,21 @@ package nutanix
 
 import (
 	"fmt"
-	"strconv"
-
-	"github.com/terraform-providers/terraform-provider-nutanix/utils"
-
 	"github.com/hashicorp/terraform/helper/schema"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	"github.com/terraform-providers/terraform-provider-nutanix/utils"
+	"strconv"
 )
 
 func dataSourceNutanixCluster() *schema.Resource {
 	return &schema.Resource{
 		Read: dataSourceNutanixClusterRead,
 		Schema: map[string]*schema.Schema{
 			"cluster_id": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type: schema.TypeString,
+				// Required: true,
```

Comment:
Could you remove this attribute and add Computed: true.

Reply:
also add `ConflictsWith: []string{"cluster_name"}` in order to raise an error when both values are set.